includes/commands/class-creative-generate.php

```diff
@@ -58,7 +58,7 @@ public function __invoke( $args, $assoc_args ) {
 		for ( $i = 1; $i <= $assoc_args['count']; $i++ ) {
 
 			$creatives[] = affwp_add_creative( array(
-				'name'        => 'Generated Creative',
+				'name'        => sprintf( 'Generated Creative %1$s', $i ),
```

Comment:
```suggestion
				'name'        => sprintf( 'Generated Creative %s', $i ),
```

I would just use `%s` to keep the translation file simple, it's easier to mess up a positional placeholder.